metaci/api/views/test_testmethod_perf.py

```diff
@@ -0,0 +1,335 @@
+from metaci.conftest import StaffSuperuserFactory, TestResultFactory
+
+from urllib.parse import urlencode
+import random
+from datetime import timedelta, datetime
+from django.utils import timezone
+
+from rest_framework.test import APIClient, APITestCase
+
+
+class _TestingHelpers:
+    def debugmsg(self, *args):
+        print(*args)  # Pytest does useful stuff with stdout, better than logger data
+
+    @classmethod
+    def make_user_and_client(cls):
+        user = StaffSuperuserFactory()
+        client = APIClient()
+        client.force_authenticate(user)
+        response = client.get("/api/")
+        if response.status_code == 400 and "DisallowedHost" in str(response.content):
+            cls.debugmsg("**** YOU MAY NEED TO ADD AN ALLOWED_HOSTS TO YOUR TEST.PY")
```

Comment:
I did add ALLOWED_HOSTS to the settings for tests, so I'm not sure we need this check now.

Reply:
Fixed in: 

https://github.com/SFDO-Tooling/MetaCI/pull/380/commits/beaf7c37b45bc92fadabf61b899dfb0faaedd09b